addons/test_mail/tests/test_performance.py

```diff
@@ -476,7 +476,7 @@ def test_mail_composer_form_attachments(self):
                 composer_form.attachment_ids.add(attachment)
             composer = composer_form.save()
 
-        with self.assertQueryCount(admin=52, employee=52):  # tm 51/51
+        with self.assertQueryCount(admin=53, employee=53):  # tm 52/52
```

Comment:
Note: this query count is incremented due to the addition of the `thumbnail` field on `ir.attachment`